plonky2/examples/poseidon2_bn254_example.rs

```diff
@@ -69,5 +69,4 @@ fn main() -> Result<()> {
         print_hash_statistics: HashStatisticsPrintLevel::Summary,
     };
     data.verify_with_options(proof, &verifier_opts)
-
```

Comment:
I would rename this file to something like `bn254_fibonacci.rs`, because it's still the fibonacci example, not poseidon2 example, it's just using a different "backend"